test/libyul/ewasmTranslationTests/balance.yul

```diff
@@ -0,0 +1,9 @@
+{
+  sstore(0, balance(address()))
+  sstore(1, balance(0))
+}
+// ----
+// Trace:
+// Memory dump:
+//      0: 0000000000000000000000000000000011111111000000000000000000000000
```

Comment:
looks wrong.

Reply:
but it may be just the interpreter.

Reply:
`balance` is not implemented yet by the translator